.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -300,36 +300,32 @@ jobs:
           awk '/^Failed Tests|Unexpectedly Passed Tests|Unresolved tests|Testing Time/{flag=1}/FAILED: CMakeFiles/{flag=0}flag' e2e.log >> $GITHUB_STEP_SUMMARY
         fi
         exit $exit_code
-    - name: List excluded SYCL CTS categories
-      if: inputs.tests_selector == 'cts' && env.CTS_TESTS_TO_BUILD == ''
-      run: |
-        if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_OCL_CPU
-        fi
-        if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_L0_GPU
-        fi
     - name: Build SYCL CTS tests
       if: inputs.tests_selector == 'cts'
       env:
         CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
       run: |
-        # Do not exclude any category if passing the CTS_TESTS_TO_BUILD variable.
         cts_exclude_filter=""
+        # If CTS_TESTS_TO_BUILD is null - use filter
         if [ -z "$CTS_TESTS_TO_BUILD" ]; then
           if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
             cts_exclude_filter=$PWD/devops/cts_exclude_filter_OCL_CPU
-          fi
-          if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
+          elif [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
             cts_exclude_filter=$PWD/devops/cts_exclude_filter_L0_GPU
           fi
+
+          # List excluded SYCL CTS categories:
+          # SYCL_CTS_EXCLUDE_TEST_CATEGORIES - Optional file specifying a list
+          # of test categories to be excluded from the build.
+          echo "::group::Excluded test categories"
+          cat $cts_exclude_filter
+          echo "::endgroup::"
         fi
 
         cmake -GNinja -B./build-cts -S./khronos_sycl_cts -DCMAKE_CXX_COMPILER=$(which clang++) \
         -DSYCL_IMPLEMENTATION=DPCPP \
         -DSYCL_CTS_EXCLUDE_TEST_CATEGORIES="$cts_exclude_filter" \
         -DSYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS=OFF \
-        -DSYCL_CTS_MEASURE_BUILD_TIMES=ON \
```

Comment:
Why are we dropping this?

Reply:
Mentioned this in the commit message (hope I'm not wrong)
> Also remove DSYCL_CTS_MEASURE_BUILD_TIMES. It affects the performance, but we don't use these results.

Reply:
@aelovikov-intel should I remove it?

Reply:
If you think it's better without it, then dropping it is fine. You're the one working with the CTS runs in CI.

Reply:
Okay, then I'd prefer to drop it